docker/docker-compose.yml

```diff
@@ -1,3 +1,4 @@
+  
```

Comment:
can you remove that newline at the first line of the file, so that the changes in the `docker-compose.yml` will disappear.